src/Microsoft.ML.Data/Transforms/ColumnBindingsBase.cs

```diff
@@ -14,16 +14,44 @@
 
 namespace Microsoft.ML.Data
 {
-    internal abstract class SourceNameColumnBase
+    /// <summary>
+    /// Specifies input and output column names for a transformation
+    /// </summary>
+    public class OneToOneColumn
     {
+        /// <summary>Name of the column resulting from the transformation of <see cref="InputColumnName"/>.</summary>
         [Argument(ArgumentType.AtMostOnce, HelpText = "Name of the new column", ShortName = "name")]
```

Comment:
Maybe add `Name="Name"` to avoid EP changes.

Reply:
I don't think they are case sensitive. So this should be fine!

---
In reply to: [267456175](https://github.com/dotnet/machinelearning/pull/3033#discussion_r267456175) [](ancestors = 267456175)

Reply:
Reactivating. What @wschin is pointing out, correctly, is that by doing this the name of the entry-point node, and the command line, will change, which seems like an unnecessary change.

---
In reply to: [267456928](https://github.com/dotnet/machinelearning/pull/3033#discussion_r267456928) [](ancestors = 267456928,267456175)